recipes/jbig/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "2.1" %}
+
+package:
+    name: jbig
+    version: {{ version }}
+
+source:
+    fn: jbigkit-{{ version }}.tar.gz
+    url: https://www.cl.cam.ac.uk/~mgk25/jbigkit/download/jbigkit-{{ version }}.tar.gz
+    md5: ebcf09bed9f14d7fa188d3bd57349522
+    patches:
+        # Remove -Wno-unused-result (for compatibility with old gcc), and add -fPIC (to allow shared libraries to use this build).
+        - Makefile.patch
+
+build:
+    number: 0
+    skip: True  # [win]
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libjbig.a  # [unix]
+        - test -f ${PREFIX}/lib/libjbig85.a  # [unix]
+        - conda inspect linkages -n _test jbig  # [linux]
+
+about:
+    home: https://www.cl.cam.ac.uk/~mgk25/jbigkit
+    license: GPL-2.0+
```

Comment:
Maybe drop the `+`. Most people familiar with the GPL know that they have such a clause.